benchmark/buffers/buffer-base64-decode.js

```diff
@@ -12,8 +12,8 @@ function main({ n, size }) {
   // eslint-disable-next-line node-core/no-unescaped-regexp-dot
   s.match(/./);  // Flatten string.
   assert.strictEqual(s.length % 4, 0);
-  const b = Buffer.allocUnsafe(s.length / 4 * 3);
-  b.write(s, 0, s.length, 'base64');
+  const b = Buffer.allocUnsafe(size * 3);
+  b.write(s, 0, size * 3, 'base64');
```

Comment:
This is a bit brittle but it should be fine: in case `s` would be changed, these two would also have to change now. That was not the case before.

Reply:
Introduced a new variable to hopefully make it less brittle without sacrificing readability.